docs/content/error/$http/baddata.ngdoc

```diff
@@ -0,0 +1,16 @@
+@ngdoc error
+@name $http:baddata
+@fullName Bad JSON Data
+@description
+
+The default @{link ng$http#default-transformations `transformResponse`} will try to parse the
+response as JSON if the `Content-Type` header is `application/json` or the response looks like a
+valid JSON-stringified object or array.
+This error occurs when that data is not a valid JSON object.
+
+The error message should provide additional context such as the actual response.
+
+To resolve this error, make sure you pass a valid JSON data object to `transformResponse`.
```

Comment:
_a valid JSON data object_ --> _valid JSON data_

Also, the correct resolution depends on the source of the problem. In some cases, one might need to send correct headers (to not include `application/json` for example).